src/metaprob/prelude.cljc

```diff
@@ -45,6 +46,10 @@
   (gen [f l]
     (doall (map-indexed (fn [i x] (at i f x)) l))))
 
+(def map-xform
```

Comment:
Will this be lazy by default? Where is it used?

Reply:
This is vestigial from an attempt we made to improve performance. It'll be removed.